website/src/pages/en/subgraphs/explorer.mdx

```diff
@@ -2,93 +2,113 @@
 title: Graph Explorer
 ---
 
-Unlock the world of Subgraphs and network data with [Graph Explorer](https://thegraph.com/explorer).
+Use [Graph Explorer](https://thegraph.com/explorer) and take full advantage of its core features.
 
 ## Overview
 
-Graph Explorer consists of multiple parts where you can interact with [Subgraphs](https://thegraph.com/explorer?chain=arbitrum-one), [delegate](https://thegraph.com/explorer/delegate?chain=arbitrum-one), engage [participants](https://thegraph.com/explorer/participants?chain=arbitrum-one), view [network information](https://thegraph.com/explorer/network?chain=arbitrum-one), and access your user profile.
+This guide explains how to use [Graph Explorer](https://thegraph.com/explorer) to quickly discover and interact with Subgraphs on The Graph Network, delegate GRT, view participant metrics, and analyze network performance.
 
-## Inside Explorer
+> When you visit Graph Explorer, you can also access the link to [explore Substreams](https://substreams.dev/).
 
-The following is a breakdown of all the key features of Graph Explorer. For additional support, you can watch the [Graph Explorer video guide](/subgraphs/explorer/#video-guide).
+## Prerequisites
 
-### Subgraphs Page
+- A wallet (e.g., MetaMask) connected to [Graph Explorer](https://thegraph.com/explorer).
+  > Make sure your wallet is connected to the correct network (e.g., Arbitrum). Features and data shown are network specific.
```

Comment:
Just as a note, users can view Graph Explorer without connecting, or even having a wallet. It is only needed at the point that they want to perform an action (like delegate, signal, etc)